src/api.js

```diff
@@ -214,6 +214,17 @@ var addUser = function (credentials, callback) {
   });
 };
 
+exports.createDefaultUser = function(credentials,callback) {
```

Comment:
Can you create your function first with a variable then export it ? Check forEachWorkspace for example.